styles.css

```diff
@@ -0,0 +1,323 @@
+.top {
+    display: flex;
+
+    height: 65px;
+    background: #FFFFFF;
+    box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.16);
+    /* justify-content: space-around; */
+    /* margin-bottom: 52px; */
+
+}
+
+.body {
+
+    margin-left: 0px;
+    margin-right: 0px;
+    padding-left: 0px;
+    padding-right: 0px;
+
+
+}
+
+.ROSE {
+
+    margin: 24px 0px 24px 20px;
+
+
+}
+
+.wholeBurger {
+
+    margin: 19px 1px 10px 70px;
+}
+
+.topPartOfTheBurger {
+
+    width: 24px;
+    height: 0px;
+    top: 39.5px;
```

Comment:
do not use fractional values for pixels. even though it is valid it doesn't make much sense. what is 0.5 pixel?